x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/text_expansion_callout.tsx

```diff
@@ -221,15 +312,28 @@ export const ModelDeployed = ({
 
 export const TextExpansionCallOut: React.FC<TextExpansionCallOutProps> = (props) => {
   const { dismiss, isDismissable, show } = useTextExpansionCallOutData(props);
-  const { isCreateButtonDisabled, isModelDownloadInProgress, isModelDownloaded } =
-    useValues(TextExpansionCalloutLogic);
+  const {
+    isCreateButtonDisabled,
+    isModelDownloadInProgress,
+    isModelDownloaded,
+    isModelStarted,
+    isStartButtonDisabled,
+  } = useValues(TextExpansionCalloutLogic);
 
   if (!show) return null;
 
   if (!!isModelDownloadInProgress) {
```

Comment:
nitpick(non-blocking): You don't need the double `!!` here. Ifs are truthy/falsy not strict Boolean checks.